### ISSUE TITLE
what's .tfci file and how to get the real codewords of an image

### ISSUE THREAD

**Issue by wxrzz** [tensorflow/compression#129]
Hello Dr. Balle,
     I have tried to use bls2017.py to train my own model, but I'm not sure that whether bpp is estimated or not when training the model. So I printed the "bits" in "y_hat, bits = entropy_model(y, training=training)  line 115", but found the numeric value (like 17400.3, I guess it is the number of bits required for encoding an image). 
    Also, I want to know what is  the .tfci file and how can I get the stream of transmitted bits after a picture is compressed, please tell me how to do when you're available, thanks!

**Comment by jonycgn** [comment]
Yes, that's correct. `bits` is the number of bits required to encode one image. Bits per pixel are computed [in this line](https://github.com/tensorflow/compression/blob/master/models/bls2017.py#L116).

.tfci is just a file extension we chose to write the compressed images to. You can write/read them with the same Python script.

The compressed stream is returned by the `compress` method [in this line](https://github.com/tensorflow/compression/blob/master/models/bls2017.py#L176). It's the first return value, followed by the image and latent shapes.
